x-pack/plugins/case/server/client/cases/create.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Boom from 'boom';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+
+import { flattenCaseSavedObject, transformNewCase } from '../../routes/api/utils';
+
+import {
+  CasePostRequestRt,
+  throwErrors,
+  excess,
+  CaseResponseRt,
+  CaseResponse,
+} from '../../../common/api';
+import { buildCaseUserActionItem } from '../../services/user_actions/helpers';
+import {
+  getConnectorFromConfiguration,
+  transformCaseConnectorToEsConnector,
+} from '../../routes/api/cases/helpers';
+
+import { CaseClientCreate, CaseClientFactoryArguments } from '../types';
+
+export const create = ({
+  savedObjectsClient,
+  caseService,
+  caseConfigureService,
+  userActionService,
+  request,
+}: CaseClientFactoryArguments) => async ({ theCase }: CaseClientCreate): Promise<CaseResponse> => {
```

Comment:
nit -> can we have just case instead of `theCase`

Reply:
I would like to but unfortunately `case` is a JS reserved word.